sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Adds support to run queries with group by over a column with null values.  [#22345](https://github.com/Azure/azure-sdk-for-js/pull/22345)
```

Comment:
`Adds` -> `Added`

Reply:
Hmm - is a mix and match in the entire file with a majority being in the now vs. past

Reply:
oh I see, I was following the java format